api/package-lock.json

```diff
@@ -1,10191 +1,8 @@
 {
   "name": "api",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Please rollback this lock file as we're not introducing any new dependencies with this feature.

Reply:
How do I do that

Reply:
ive merged with main and accepted main's version of the file, so should be fine now